developerportal/templates/molecules/cards/card-video.html

```diff
@@ -8,12 +8,17 @@
 
 <a
   href="{% pageurl resource %}"
-  class="card-link {% if resource.is_external %}js-modal-trigger{% endif %}"
-  data-type="{{ resource.resource_type }}"
+  class="card-link {% if filter_target %}js-filter-target{% endif %} {% if resource.is_external %}js-modal-trigger{% endif %}"
   {% if resource.is_external %}
     data-class-name="mzp-has-media"
     data-title="{{ resource.title }}"
   {% endif %}
+  data-type="{{ resource.resource_type }}"
+  {% if filter_target %}
+    data-topics="{% for topic in resource.topics.all %}{{ topic.topic.slug }} {% endfor %}"
+    data-month="{{ resource.month_group|date:'Y-m' }}"
+    hidden
+  {% endif %}
```

Comment:
This extra markup came from `card-article.html` to ensure both types of content are treated the same way by the filters